src/models/FileMetada.ts

```diff
@@ -0,0 +1,11 @@
+export type FileMetada = {
+    av_status: 'scanned' | 'not-scanned';
+    content_type: string;
+    id: string;
+    links: {
```

Comment:
Shall we get rid of this? We take control of how resource is being download so it becomes needed.

Reply:
remove links?

Reply:
Yeh, we will never use it.